WebCryptoAPI/derive_bits_keys/hkdf.js

```diff
@@ -0,0 +1,294 @@
+
+function run_test() {
+    // May want to test prefixed implementations.
+    var subtle = self.crypto.subtle;
+
+    // hkdf2_vectors sets up test data with the correct derivations for each
+    // test case.
+    var testData = getTestData();
+    var passwords = testData.passwords;
```

Comment:
You don't have to fix, but i would prefer a different term here than password.  HKDF is only marginally defined to be a password based KDF function.  PBKDF is better for that.  Not misleading people is always good.